src/mcts/node.h

```diff
@@ -340,6 +344,27 @@ class EdgeAndNode {
   float GetD() const {
     return (node_ && node_->GetN() > 0) ? node_->GetD() : 0.0f;
   }
+  float GetVariance(float default_var) const {
+    if (node_ && node_->GetN() > 1) {
+      return node_->GetSquaredDiff() / (node_->GetN() - 1);
+    }
+    return default_var;
+  }
+  float GetQLCB() const {
+    if (!node_) {
+      return -1e6f;
+    }
+    auto visits = node_->GetN();
+    if (visits < 2) {
+      // Return large negative value if not enough visits.
+      return -1e6f + visits;
+    }
+
+    auto stddev = std::sqrt(GetVariance(1.0f) / visits);
```

Comment:
Technically this is the standard error (standard deviation divided by sqrt(N)). Maybe `stderr` is a better variable name.

Or is this the standard deviation of something in another context?

Reply:
I was thinking of the estimated distribution for the mean which has this standard deviation. Standard error seems to be correct too.

Reply:
Okay, that seems logical, I didn't think about it that way :).